ads/_config.js

```diff
@@ -522,6 +522,12 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'invibes': {
+    prefetch: 'https://k.r66net.com/GetAmpLink',
+    renderStartImplemented: true,
```

Comment:
Thanks for implementing the renderStart API. When I was testing with the example you provide, I didn't see the render-start message. Could you please double check that the API is implemented correctly. Thank you

Reply:
Our GetAmpLink endpoint is not production ready. It's in testing and should be ready within the week. The example is with a test placement '1234' and only redirects to a basic script/creative which renders an image logo.
The script which will be deployed next week is at
http://static.videostepstage.com/scripts/amp.min.js

The renderStart is implemented like this
`
var _ampContext = window.context;

invibes.addEventListener('ivbsAdRendered', function () {
  _ampContext.renderStart();
);

invibes.addEventListener('ivbsFallback', function () {
  _ampContext.noContentAvailable();
});
`

The 'ivbsAdRendered' event is dispatched if we have an ad available from our ad server, certain context conditions are met etc..
Else the 'ivbsFallback' event is dispatched.